.github/workflows/run-tests.yml

```diff
@@ -31,9 +35,12 @@ jobs:
           php-version: ${{ matrix.php }}
           extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlit3, pdo_sqlite
```

Comment:
`sqlite3` is misspelled here, which is why one of the errors is showing up in the logs

Reply:
Thanks.